aeppl/abstract.py

```diff
@@ -69,16 +69,18 @@ def assign_custom_measurable_outputs(
 
     new_node = node.clone()
     op_type = type(new_node.op)
+
+    # TODO: The above could be a stand-alone utility function for all sorts of
+    # instance-based dispatching
```

Comment:
What does this TODO comment mean?

Reply:
You can remove this; it's an old TODO that no longer applies.